src/zenml/zen_stores/schemas/artifact_schemas.py

```diff
@@ -266,12 +268,14 @@ def from_request(
             data_type=artifact_version_request.data_type.json(),
         )
 
-    def to_model(self, hydrate: bool = False) -> ArtifactVersionResponse:
+    def to_model(
+        self, include_metadata: bool = False, include_resources: bool = False
+    ) -> ArtifactVersionResponse:
         """Convert an `ArtifactVersionSchema` to an `ArtifactVersionResponse`.
 
         Args:
-            hydrate: bool to decide whether to return a hydrated version of the
-                model.
+            include_metadata: Whether the metadata will be filled.
+            include_resources: Whether the metadata will be filled.
```

Comment:
```suggestion
            include_resources: Whether the resources will be filled.
```

I won't suggest this for other schemas, but they all have the same issue

Reply:
will fix everywhere